docs/variants/hardkernel_odroid_h4_plus/releases.md

```diff
@@ -0,0 +1,17 @@
+# Dasharo compatible with Hardkernel ODROID H4+
+
+Following Release Notes describe status of Open Source Firmware development for
+Hardkernel ODROID H4+
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+
+[Subscribe to Hardkernel ODROID H4+ Dasharo Release Newsletter]
+[newsletter]{ .md-button .md-button--primary .center }
+
+</center>
+
+<!-- TODO -->
+[newsletter]: ./releases.md
```

Comment:
Trzeba utworzyć link zapisu do newslettera i dodać tutaj.

Reply:
Newsletter is alrady added: https://newsletter.3mdeb.com/subscription/pULA4K0Eo

Reply:
https://github.com/Dasharo/docs/pull/903/commits/e15fa98fc9e9d74b002b51bf19e762e0e71db3e3